Networking/Networking/Model/Refund/Refund.swift

```diff
@@ -10,17 +10,17 @@ public struct Refund: Codable {
     public let dateCreated: Date // gmt
     public let amount: String
     public let reason: String
-    public let refundedByUserID: Int
```

Comment:
I noticed when thinking of `Refund`, that I made redundant names. I can't tell if `Refund.byUserID` is any better or not though, so I will change it back if you think the older version is better.

Reply:
I slightly preferred `refundedByUserID` since it tells the action of the user, but either way is good :)